examples/offscreen-canvas.js

```diff
@@ -184,7 +183,7 @@ const info = document.getElementById('info');
 function showInfo(propertiesFromFeatures) {
   if (propertiesFromFeatures.length == 0) {
     info.innerText = '';
-    info.style.opacity = 0;
+    info.style.opacity = String(0);
```

Comment:
```suggestion
    info.style.display = 'none';
```

Reply:
Opacity is used because there is a 100ms transition.

Reply:
Oh right, that was an oversight on my part. So then `'0'` instead of `String(0)` would look nicer.